dressup/src/pages/ProductsPage/FilterSection/FilterSection.js

```diff
@@ -27,6 +32,9 @@ const FilterSection = () => {
               type="radio"
               name="sort-by"
               value="Price low to high"
+              onChange={() =>
+                dispatch({ type: "lowToHigh", value: "lowToHigh" })
+              }
             ></input>
             <label htmlFor="Price low to high">Price low to high</label>
```

Comment:
Suggestion: provide `id` attribute with same value as `htmlFor` for connecting them